dev-python/markuppy/markuppy-1.1.4.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
+
+inherit distutils-r1
+
+DESCRIPTION="Format-agnostic tabular dataset library"
+HOMEPAGE="https://pypi.org/project/MarkupPy/ https://github.com/tylerbakke/MarkupPy"
+SRC_URI="https://files.pythonhosted.org/packages/4e/ca/f43541b41bd17fc945cfae7ea44f1661dc21ea65ecc944a6fa138eead94c/MarkupPy-1.14.tar.gz -> ${P}.tar.gz"
+
+SLOT="0"
+LICENSE="BSD-2"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+
+# MarkupPy does not have any test suite
+RESTRICT="test"
```

Comment:
Are you sure packaging it is a good idea? Given that upstream didn't manage to actually push code into the repository, and didn't do anything for 3.5 years about it, I feel like packaging it is a waste of time.

Reply:
tablib is required for newer versions of netbox, this is why I'm packaging it